static/js/popovers.js

```diff
@@ -908,15 +908,11 @@ export function register_click_handlers() {
     });
 
     $("body").on("click", ".info_popover_actions .narrow_to_private_messages", (e) => {
-        const user_id = elem_to_user_id($(e.target).parents("ul"));
-        const email = people.get_by_user_id(user_id).email;
         hide_all();
         if (overlays.settings_open()) {
             overlays.close_overlay("settings");
         }
-        narrow.by("pm-with", email, {trigger: "user sidebar popover"});
```

Comment:
I'm not familiar enough with the narrowing code to tell if this change breaks anything or makes it slower, etc. It seems to work correctly, though.  @timabbott might have comments on this. 

Meta comments on the commit message: 
- I think `Partially Fixes #XXX` is treated the same as `Fixes #XXX` by GitHub, unless this has been changed recently.  
- The commit message summary could better explain what the commit is doing i.e., simplifying the click handler. View private messages would correctly open in new tab even without this change.
- Also, commit message body is too long and can be wrapped to 68 chars (as per the recommendation in Zulip docs)

Reply:
@punchagan , I updated the PR with the changes requested in commit messages.
would like you to please take a look at them.

Reply:
Talking about the other elements which need this sort of fix -- 

Till now I've found 1 more area which needs this fix -- `Users List` in right-sidebar, I confirmed with Alya for this. [this](https://chat.zulip.org/#narrow/stream/9-issues/topic/Ctrl.2BClick.20to.20open.20in.20new.20tab/near/1403557)

Reply:
This `trigger` parameter is part of a system for logging which buttons were used to get a user to a given narrowed view. So we shouldn't remove this.

I think we can check whether the event is a ctrl+click, and if so, just return, to make this work as desired.